wowup-electron/src/app/components/options/wow-client-options/wow-client-options.component.ts

```diff
@@ -82,12 +82,14 @@ export class WowClientOptionsComponent implements OnInit, OnDestroy {
         return "assets/images/wow-classic-logo.png";
       case WowClientType.Retail:
       case WowClientType.RetailPtr:
+      case WowClientType.RetailXPtr:
       case WowClientType.Beta:
         return "assets/images/wow-dragonflight-logo.png";
       case WowClientType.ClassicPtr:
       case WowClientType.Classic:
-      case WowClientType.ClassicBeta:
         return "assets/images/wow-classic-wotlk-logo.png";
+      case WowClientType.ClassicBeta:
+        return "assets/images/wow-classic-cataclysm-logo.png";
```

Comment:
I suspect all classic variants should in theory be cata now

Reply:
Good call